frequently_occurring.py

```diff
@@ -0,0 +1,26 @@
+'''В большой текстовой строке подсчитать количество встречаемых слов и вернуть 10 самых частых.
+Не учитывать знаки препинания и регистр символов.
+За основу возьмите любую статью из википедии или из документации к языку.'''
+from collections import Counter
+import string
+import wikipedia
+
+NUMBER_WORDS = 10
+TITLE_ARTICLE = "Россия"
+
+print(wikipedia.summary(TITLE_ARTICLE))
+
+text = wikipedia.summary(TITLE_ARTICLE)
+# print(f'текст{text}')
+text_low = text.lower()
+
+text_clear = text_low.translate(str.maketrans('', '', string.punctuation))
+# print(f'\n{text_clear}')
+text_list = text_clear.split(' ', )
```

Comment:
Если хотите разделить текст по пробелам, то лучше ничего не передавать методу `split`.
```suggestion
text_list = text_clear.split()
```